kentikapi/client_integration_test.go

```diff
@@ -417,3 +425,110 @@ func timePtr(v time.Time) *time.Time {
 func durationPtr(v time.Duration) *time.Duration {
 	return &v
 }
+
+func TestGRPCClient_GetAgent(t *testing.T) {
+	tests := []struct {
+		name    string
+		id      string
+		res     *pb.GetAgentResponse
+		errCode codes.Code
+		errMsg  string
+	}{
```

Comment:
Please also add "empty request, status InvalidArgument received" case, to test that our code does not panic when empty request data is passed.